composer.json

```diff
@@ -11,22 +11,25 @@
             "email": "dcramer@gmail.com"
         }
     ],
-    "require-dev": {
-        "friendsofphp/php-cs-fixer": "~2.1",
-        "phpunit/phpunit": "^4.8 || ^5.0",
-        "ext-openssl": "*",
-        "nokitakaze/testhttpserver": "^0.1.0",
-        "monolog/monolog": "~1.0"
-    },
     "require": {
-        "php": ">=5.5",
-        "ext-hash": "*",
+        "php": ">=5.6",
```

Comment:
@ste93cry nope, `>=5.6` would allow `8.0` for example. We don't want that, or in the future we will disallow that and composer will fall back to a very old, not really compatible version. It's better to be conservative on this side of thing, allowing 8.0 later is easy, if no issue is found.

@stayallive I think that testing on 7.2 in advance is good, but we don't need to `--ignore-platform-reqs`, I was suggesting `^5.6 | ^7.0` which allows all the 7.x series; I would only put the 7.2 build on `alllow_failure` for now.

For HHVM,  I don't know if I would still be testing on that, since a lot of other big players are leaving it, especially Symfony, from which we use a component, that could then break on HHVM leaving us without a solution. Maybe we should put it on `allow_failure` too?